packages/pull-signal/src/repositories/delegation.repository.ts

```diff
@@ -0,0 +1,44 @@
+import { DB, TableName, genericError } from "pagopa-signalhub-commons";
+
+import { config } from "../config/env.js";
+
+export interface IDelegationRepository {
+  findBy: (
+    delegateId: string,
+    eServiceId: string
```

Comment:
```suggestion
    eServiceId: string,
    state: string
```    
Like is [other operations](export interface IInteropRepository {) on Repository: we pass state as parameter 
    
```